modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,140 @@
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role. For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
+
+. Install the {TempoOperator}:
+
+.. Go to *Operators* -> *OperatorHub* and search for `{TempoOperator}`.
+
+.. Select the *{TempoOperator}* that is *OpenShift Operator for Tempo* -> *Install* -> *Install* -> *View Operator*.
++
+[IMPORTANT]
+====
+This installs the Operator with the default presets:
+
+* *Update channel* -> *stable*
+* *Installation mode* -> *All namespaces on the cluster*
+* *Installed Namespace* -> *openshift-tempo-operator*
+* *Update approval* -> *Automatic*
+====
+
+.. In the *Details* tab of the page of the installed Operator, under *ClusterServiceVersion details*, verify that the installation *Status* is *Succeeded*.
+
+. Create a Secret for your object storage bucket: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Create an OpenShift project of your choice for the *TempoStack* instance that you will create in the next step: go to *Home* -> *Projects* -> *Create Project*.
+
+. Create a *TempoStack* instance.
++
+NOTE: You can create multiple *TempoStack* instances in separate projects on the same cluster.
+
+.. Go to *Operators* -> *Installed Operators*.
++
+[TIP]
+====
+You can use the *Project:* dropdown list to select the OpenShift project that you created for the *TempoStack* instance in the previous step.
+====
+
+.. Select *TempoStack* -> *Create TempoStack* -> *YAML view*.
+
+.. In the *YAML view*, customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <openshift_project_of_tempostack_instance>
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: <secret-name> <1>
+      type: <secret-provider> <2>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The value of the `name` in the `metadata` of the Secret.
+<2> The accepted values are `azure` for Azure Blob Storage, `gcs` for Google Cloud Storage, and `s3` for Amazon S3 or MinIO or Red Hat OpenShift Data Foundation.
++
+.TempoStack CR for AWS S3 and MinIO storage
+====
```

Comment:
I think we want to avoid this, since it seems to separate the title from the block. I don't know a hard rule on this, but this is the first time I have ever seen it in docs, so I would lean towards staying consistent in presentation. You can still have the line that follows it attach to the block inline under it without the box.

Reply:
This is a standard AsciiDoc formatting for examples.

If you have `.Secret for Amazon S3 and MinIO storage ====`, the heading will be rendered as `Example 5. Secret for Amazon S3 and MinIO storage`.

If you have `.Secret for Amazon S3 and MinIO storage ----`, the heading will be rendered as `Secret for Amazon S3 and MinIO storage`.

Anyway, I've removed the `====` syntax from all of my examples here that don't have introductory text.